packages/opentelemetry-context-async-hooks/test/AsyncHooksContextManager.test.ts

```diff
@@ -416,6 +452,26 @@ for (const contextManagerClass of [
         assert.strictEqual(patchedEe.listeners('test').length, 1);
         patchedEe.emit('test');
       });
+
+      it('should not influence other instances', () => {
+        const ee = new EventEmitter();
+        otherContextManager = new contextManagerClass();
+        otherContextManager.enable();
+
+        const context = ROOT_CONTEXT.setValue(key1, 2);
+        const otherContext = ROOT_CONTEXT.setValue(key1, 3);
+        const patchedEe = otherContextManager.bind(
```

Comment:
```suggestion
        const patchedEE = otherContextManager.bind(
```

Reply:
All other tests use also `patchedEe`. Should I update all?

Reply:
`patchedEe` stands for `patched Event Emitter` right ?

Reply:
I assume so yes, as mentioned just copied from tests above...

Reply:
personally I would change it, it will look better, but not a blocker

Reply:
done